lms/envs/common.py

```diff
@@ -1914,6 +1899,22 @@ def _make_locale_paths(settings):
 # let logging work as configured:
 CELERYD_HIJACK_ROOT_LOGGER = False
 
+############################## HEARTBEAT ######################################
+
+# Checks run in normal mode by the heartbeat djangoapp
+HEARTBEAT_CHECKS = [
+    'openedx.core.djangoapps.heartbeat.default_checks.check_modulestore',
+    'openedx.core.djangoapps.heartbeat.default_checks.check_database',
+]
+
+# Other checks to run by default in "extended"/heavy mode
+HEARTBEAT_EXTENDED_CHECKS = (
+    'openedx.core.djangoapps.heartbeat.default_checks.check_celery',
+)
+
+HEARTBEAT_CELERY_TIMEOUT = 5
+CELERY_CHECK_ROUTING_KEY = HIGH_PRIORITY_QUEUE
```

Comment:
@alanoe @kaizoku I'm seeing these errors in the logs of a newly-provisioned instance (cf [SE-2411](https://tasks.opencraft.com/browse/SE-2411)) , and it's making the `edxapp_worker:cms_default_1` service fail to start, because this new setting is missing from the `cms.envs`.

```
==> /edx/var/log/supervisor/cms_default_1-stderr.log <==
  File "/edx/app/edxapp/edx-platform/openedx/core/djangoapps/heartbeat/tasks.py", line 8, in <module>
    @task(routing_key=settings.CELERY_CHECK_ROUTING_KEY)
  File "/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/django/conf/__init__.py", line 57, in __getattr__
    val = getattr(self._wrapped, name)
AttributeError: 'Settings' object has no attribute 'CELERY_CHECK_ROUTING_KEY'
```

Looks like the usual method is to add it to [this list of variables imported from `lms.envs.common`](https://github.com/open-craft/edx-platform/blob/9a50786e8db278e1a3257c3648524b77c7ed5a34/cms/envs/common.py#L51), but I'm not positive that gets the same `ENV_TOKEN` overrides, so will need testing.

Reply:
BTW, this is urgent, as this branch is used to deploy our production instances, and so is breaking them too.